specification.md

```diff
@@ -62,15 +53,15 @@ The `OrderSide` type is an enumerated type with the following fields:
 ### `Order` ###
 #### Description ####
 
-The `Order` type represents an order in a Tracer market.
+The `Order` type represents an order in a Tracer Perpetual Swaps market.
 
 #### Fields ####
 
 | Name | Type | Description |
 | ---- | ---- | ----------- |
-| ID | 256-bit unsigned integer | The unique identifier of the order |
+| ID | 32-byte Keccak-256 digest | The unique identifier of the order |
 | Address | Ethereum address | The Ethereum address of the trader submitting this order |
-| Market | Ethereum address | The Ethereum address of the Tracer market |
+| Market | Ethereum address | The Ethereum address of the Tracer Perpetual Swaps market |
 | Side | `OrderSide` | The side of the market for the order |
 | Price | 256-bit unsigned integer | The price of the order |
 | Amount | 256-bit unsigned integer | The quantity of the order |
```

Comment:
order creation field is needed. Maybe an example json input of this in the `submit` function too just to clarify field names (eg no capitals)

Reply:
Consult 9e3b09f.